iidm/iidm-api/src/main/java/com/powsybl/iidm/network/events/PropertyUpdateNetworkEvent.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.network.events;
+
+import java.util.Objects;
+
+/**
+ * @author Geoffroy Jamgotchian {@literal <geoffroy.jamgotchian at rte-france.com>}
+ */
+public record PropertyUpdateNetworkEvent(String id, String attribute, PropertyUpdateType updateType, Object oldValue, Object newValue) implements NetworkEvent {
```

Comment:
Same comment regarding `attribute` parameter, rather `key`.

And shouldn't we call this `PropertiesUpdateNetworkEvent`? Indeed, the update is not on the propert**y** (a bit strange to talk about update with a new key or a removed key), but on the propert**ies** with a specific key.

Then we should speak  of `PropertiesUpdateType`, and maybe `ADDED_PROPERTY`, `REMOVED_PROPERTY`, `UPDATED_PROPERTY`?

Reply:
Done